tripy/nvtripy/frontend/ops/pooling/avgpool.py

```diff
@@ -115,6 +58,8 @@ def avgpool(
             to apply to the input along each spatial dimension before and after the dimension respectively,
             where :math:`M` is the number of spatial dimensions, i.e. :math:`M = \text{rank(input)} - 2`.
             Defaults to all 0.
+        count_include_pad: When True, the padded values will be included in the averaging operation. This
+            option can only be ``True`` when input datatype is :class:`int8`.
```

Comment:
nit:
```suggestion
            option must be ``True`` if the input data type is :class:`int8`.
```
The current wording sounds like only `False` is allowed if the dtype is not int8